README.md

```diff
@@ -1,8 +1,60 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql)
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Anne Li . Jarvis Consulting
 
+I recently graduated from Ryerson University where I received my Bachelor of Engineering, majoring in Computer Engineering. I delved into programming during my time at university and I am fascinated by its endless possibilities in solving problems. Previously I did a 4-month internship at De Havilland, an aircraft manufacturing company, where I worked as a software engineer to help build their data-visualization tool. Currently, I am a data engineer trainee at Jarvis where I have been able to challenge myself with various projects utilizing Linux/Bash, Java 8, Docker, Maven, SpringBoot, GCP (and MORE) in a collaborative environment. I keep myself open to trying new things; whether that be my technical abilities, in personal projects, or providing quality service, I'm always eager to continuously learn to be a better engineer.
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** C, JavaScript, Python, HTML, CSS
+
+**Familiar:** Docker, GCP, Pub/Sub, REST API, Spring, Django/Flask, Matlab
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AnneLi](https://github.com/jarviscanada/jarvis_data_eng_AnneLi)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AnneLi/tree/master/linux_sql)]: This project automates the process of monitoring servers/nodes of linux that are connected through a switch and communicates internally through IPv4 addresses. Implemented bash scripts that obtain data on hardware specifications and on-going Linux resource usage data.
```

Comment:
Developed an application in bash that ...

Bash, Linux, Docker, RDBMS/PostgreSQL, SQL, crontab